.travis.yml

```diff
@@ -14,14 +18,13 @@ before_script:
   - mysql -u root -e 'CREATE DATABASE arsi_test CHARACTER SET utf8'
 
 gemfile:
- - gemfiles/rails3.gemfile
  - gemfiles/rails40.gemfile
  - gemfiles/rails41.gemfile
  - gemfiles/rails42.gemfile
+ - gemfiles/rails50.gemfile
```

Comment:
might as well add 51/52 if it's not causing too much extra work

Reply:
Will do. Wanted to settle this first and then add the files.